Karma-Khameleons/bangazon-website
# Am add line item

## Description
Add a line item. If the order hasn't been created yet, create the order.

## Related Ticket(s)
List related ticket(s) that apply to the work done in this pull request
#2

## Expected Behavior
A customer should be able to view a product's detail, add to order, and see a success page.


## Steps to Test Solution
```git fetch --all
git checkout amAddLineItem
rm -rf bang_app/migrations
rm -rf db.sqlite3
python manage.py makemigrations bang_app
python manage.py migrate
python manage.py runserver
```
* Register a new user.
* Select a product
* Add to order
* Verify a new line in the db file


## Unit Testing

## Documentation

## Deploy Notes
Follow the steps in  Steps to Test Solution.
